cmd/server/assets/realmkeys.html

```diff
@@ -0,0 +1,262 @@
+      <!-- This is only shown if the realm is NOT set to use per-realm signing keys -->
+      <div class="card mb-3">
+        <div class="card-header">
+          System verification certificate information
+        </div>      
+        <div class="card-body">
+          <div class="alert alert-info" role="alert">
+                This information needs to be communicated to the operator
+                of you exposure notification key server in order to be able to utilize
+                verification certificates issued by this server.
+          </div>
+ 
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Issuer (iss):</label>
+            <div class="col-sm-9">
+              <input type="text" id="iss" class="form-control" value="{{.certIssuer}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Audience (aud):</label>
+            <div class="col-sm-9">
+              <input type="text" id="aud" class="form-control" value="{{.certAudience}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Certificate duration:</label>
+            <div class="col-sm-9">
+              <input type="text" id="certDuration" class="form-control" value="{{.certDuration}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Key ID (kid):</label>
+            <div class="col-sm-9">
+              <input type="text" id="certKeyID" class="form-control" value="{{.certKeyID}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Public key:</label>
+            <div class="col-sm-9">
+            {{if .certPublicKey}}
+            <div class="alert alert-secondary" role="alert">
+            <pre>{{.certPublicKey}}</pre>
+            </div>
+            {{else}}
+            <div class="alert alert-danger" role="alert">{{.certPublicKeyError}}</div>
+            {{end}}
+            </div>
+          </div>
+        </div>
+      </div>
+    {{end}}
+      
+    <div class="card mb-3">
+        <div class="card-header">
+          Realm signing key configuration
+        </div>
+        <div class="card-body">
+        {{if .supportsPerRealmSigning}}            
+          {{if not $realm.UseRealmCertificateKey}}
+            {{if $realm.CanUpgradeToRealmSigningKeys}}
+            <form method="POST" action="/realm/keys/upgrade">
+                {{ .csrfField }}
+                <div class="form-group row">
+                  <div class="col-sm-12">
+                    <div class="alert alert-warning" role="alert">
+                        <strong>Unerversable action:</strong> Before upgrading to realm specific signing keys, make sure that you have
```

Comment:
UX thing:

- I think we should make the text on the page more about what happens and less about the consequences. Maybe:

> Upgrade to realm-specific signing keys. By default, realms use a globally shared signing key. Moving to realm-specific signing keys gives you more control over your keys and their rotation.
> 
> <strong>Warning!</strong> You must work with the <em>key server</em> administrator to register your new realm-specific signing key.

And then, on click, we do a javascript confirm (there's a data- helper in header.html) with a stronger warning.